dev/io.openliberty.microprofile.metrics.internal.5.x.monitor_fat/fat/src/io/openliberty/microprofile/metrics/internal/monitor_fat/MetricsMonitorTest.java

```diff
@@ -99,15 +102,30 @@ public void testVendorMetricsPresent() throws Exception {
         Assert.assertNotNull("CWWKO0219I NOT FOUND", server.waitForStringInLog("defaultHttpEndpoint-ssl", 60000));
         Log.info(c, testName, "------- server started -----");
         Assert.assertNotNull("CWWKT0016I NOT FOUND", server.waitForStringInLogUsingMark("CWWKT0016I"));
+
+        Assert.assertNotNull("No CWPMI2003I message", server.waitForStringInLogUsingMark("CWPMI2003I"));
+        
         checkStrings(getHttpsServlet("/metrics"), new String[] { "mp_scope=\"base\"", "p_scope=\"vendor\"" }, new String[] {});
 
+        
+        server.setMarkToEndOfLog();
         Log.info(c, testName, "------- Remove mpMetrics-5.0: no metrics should be available ------");
         server.setServerConfigurationFile("server_monitorOnly.xml");
-        String logMsg = server.waitForStringInLogUsingMark("CWPMI2003I");
+        Assert.assertNotNull("CWWKF0008I NOT FOUND", server.waitForStringInLogUsingMark("CWWKF0008I"));
+       
+        //Check for Trace output from MetricsConfig indicating that the WAB is unregistered (this would imply /metrics no longer active)
+        server.waitForStringInTrace("io.openliberty.microprofile.metrics50.internal.MetricsConfig 1 Unregistered web app bundle");       
+        
         
-        //TODO: actually test that /metrics does not respond
-        Log.info(c, testName, logMsg);
-        Assert.assertNotNull("No CWPMI2003I message", logMsg);
+        //For additional checking, check that we will get an ConnectException when trying to connect to /metrics
+        String exceptionString = null;
```

Comment:
Use `assertThrows()` is easier to understand but I think this works too.

Reply:
The junit used in openliberty is an older version and doesn't come with the _assertThrows()_.